lib/modules/platform/utils/read-only-issue-body.ts

```diff
@@ -2,10 +2,10 @@ import { regEx } from '../../../util/regex';
 
 export function readOnlyIssueBody(body: string): string {
   return body
-    .replace(' only once you click their checkbox below', '')
-    .replace(' unless you click a checkbox below', '')
-    .replace(' To discard all commits and start over, click on a checkbox.', '')
-    .replace(regEx(/ Click (?:on |)a checkbox.*\./g), '')
+    .replace(' only once you select their checkbox below', '')
```

Comment:
Am I making a **breaking change** here, by accident? What happens to old dashboards that still use the verb `click`?

Reply:
They should be updated in place